### PR TITLE
Feature/descw 2393 update alpha footer template

### PR DESCRIPTION
**Pull request by robrien-bcgov**
## 1.8.2 May 27, 2024

-   Update alpha footer template to match gov.bc.ca text ([DESCW-2393](https://apps.itsm.gov.bc.ca/jira/browse/DESCW-2393))

Screenshot 2024-05-28 at 10 08 26 AM

### CODE REVIEW

File: parts/footer-alpha-v3.html [hunk 1]
```diff
@@ -12,71 +12,67 @@
...
+        </a>
+    </p> <!-- /wp:paragraph --></div>
 	<!-- /wp:group --></div>
 	<!-- /wp:column -->
-	
+
 	<!-- wp:column {"width":"15%"} -->
 	<div class="wp-block-column" style="flex-basis:15%"><!-- wp:heading {"style":{"spacing":{"margin":{"top":"0","right":"0","bottom":"0","left":"0"},"padding":{"top":"0","right":"0","bottom":"var:preset|spacing|20","left":"0"}},"typography":{"fontSize":"1rem"},"elements":{"link":{"color":{"text":"var:preset|color|text-color"}}}},"textColor":"text-color"} -->
-	<h2 class="wp-block-heading has-text-color-color has-text-color has-link-color" style="margin-top:0;margin-right:0;margin-bottom:0;margin-left:0;padding-top:0;padding-right:0;padding-bottom:var(--wp--preset--spacing--20);padding-left:0;font-size:1rem">More info</h2>
+	<h2 class="wp-block-heading has-text-color-color has-text-color has-link-color" style="margin-top:0;margin-right:0;margin-bottom:0;margin-left:0;padding-top:0;padding-right:0;padding-bottom:var(--wp--preset--spacing--20);padding-left:0;font-size:1rem"><p style="font-weight: bold; font-size: 14px">MORE INFO</p>
```
Comment (ShawnTurple) [inline parts/footer-alpha-v3.html]: we should never put words all uppercase, stying should be used to transform words, if really required.

Comment (robrien-bcgov) [reply parts/footer-alpha-v3.html]: Should be good now. I even made the fontsize rem

File: parts/footer-alpha-v3.html [hunk 2]
```diff
@@ -12,71 +14,75 @@
...
-	
-	<!-- wp:social-link {"url":"https://twitter.com/BCGovNews","service":"x","className":"icon"} /--></ul>
-	<!-- /wp:social-links --></div>
-	<!-- /wp:group -->
-	
-	<!-- wp:paragraph {"align":"left","style":{"spacing":{"padding":{"top":"0.5rem","left":"0rem"}},"typography":{"fontSize":"1rem"}},"textColor":"text-color"} -->
-	<p class="has-text-align-left has-text-color-color has-text-color" style="padding-top:0.5rem;padding-left:0rem;font-size:1rem">© 2023 Government of British Columbia.</p>
+
+<!-- wp:column {"width":"15%"} -->
+<div class="wp-block-column" style="flex-basis:15%"><!-- wp:heading {"style":{"spacing":{"margin":{"top":"0","right":"0","bottom":"0","left":"0"},"padding":{"bottom":"var:preset|spacing|10"}},"elements":{"link":{"color":{"text":"var:preset|color|text-color"}}},"typography":{"fontStyle":"normal","fontWeight":"600"}},"textColor":"text-color","className":" text-uppercase","fontSize":"extra-small"} -->
```
Comment (ShawnTurple) [inline parts/footer-alpha-v3.html:32]: why did you change the font-size from rem's to normal, or did wordpress do this automatically

**Review by ShawnTurple — APPROVED** [review]

File: parts/footer-alpha-v3.html (hunk 2 above)
Comment (robrien-bcgov) [reply parts/footer-alpha-v3.html:32]: wordpress did that thru the gutenberg editor
